### DISCUSSION TITLE
How to turn of Validation?

### DISCUSSION THREAD

**Discussion by MrEWman** [JoshClose/CsvHelper#1885] [Q&A]
In the past versions of CsvHelper (ver 12) one could null out things in the CsvReader's Configuration object and read records that didn't match the ClassMap. For instance, one teammember of mine writes:

        public bool MyHeaderValidationOff { get; set; } = false;

        private void TurnOffValidation( ref CsvHelper.CsvReader csv )
        { 
           csv.Configuration.HeaderValidated = null;
           csv.Configuration.MissingFieldFound = null;
           MyHeaderValidationOff = true;
        }

...and that was enough to make CSVReader able to read records that had extra columns; however, with this new CsvHelper release (ver 27) the Configuration has private sets and readonlys now - I can't turn Validation Off like my teammember  intended.  What am I missing here? Were they using CSVReader wrong?

**Comment by JoshClose** [comment]
```cs
var config = new CsvConfiguration(CultureInfo.InvariantCulture)
{
    HeaderValidated = null,
    MissingFieldFound = null,
};
```
